packages/sentry/sentry.js

```diff
@@ -1,68 +1,83 @@
-import Raven from 'raven-js';
+import * as Sentry from '@sentry/browser';
 
 export const boot = () => {
-  if (window.app.env === 'production')
-    Raven.config(window.app.trackingSentry, {
+  if (window.app.env === 'production') {
+    Sentry.init({
+      dsn: window.app.trackingSentry,
       release: window.app.revision,
-      tags: { role: 'frontend' },
       environment: window.app.env,
       // in order to reduce the noise in sentry we only track errors that come
       // from our code and ignore errors that come from other services
       // https://blog.sentry.io/2017/03/27/tips-for-reducing-javascript-error-noise.html
       whitelistUrls: [window.app.cdnUrl, window.app.frontendHost],
-    }).install();
+    });
+    Sentry.configureScope(scope => {
+      scope.setTag('role', 'frontend');
+    });
+  }
 };
 
 export const updateUser = user => {
   if (user && window.app.env === 'production') {
     // to avoid sending personal data to sentry we anonymize the email address
     // by only sending the domain part or the email
     const emailTld = user.email.split('@')[1];
-    Raven.setUserContext({
-      email: `xxx@${emailTld}`,
-      id: user.id,
+    Sentry.configureScope(scope => {
+      scope.setUser({
+        email: `xxx@${emailTld}`,
+        id: user.id,
+      });
     });
   }
 };
 
 export const stopTrackingUser = () => {
-  if (window.app.env === 'production') Raven.setUserContext();
+  if (window.app.env === 'production')
+    Sentry.configureScope(scope => {
+      scope.clear();
+    });
 };
 
-// NOTE: only exported for tests => use reportError below for your code
-export const createSentryErrorReporter = (
-  environment,
-  debugLogger = console,
-  productionLogger = Raven
-) => (error, extraInfo) => {
-  if (error instanceof Error === false && environment === 'development') {
-    debugLogger.warn(
-      '[SENTRY]: You called "sentry.reportError" with an argument that is not an instance of "Error". ' +
+export const reportErrorToSentry = (error, extraInfo, getIsEnabled) => {
```

Comment:
We don't need the factory function anymore as we test things a bit differently now (using the sentry-testkit).